apps/web/public/static/locales/de/common.json

```diff
@@ -1221,7 +1221,7 @@
   "generate_api_key_description": "Generieren Sie einen API-Schlüssel für {{appName}} unter",
   "your_unique_api_key": "Ihr eindeutiger API-Key",
   "copy_safe_api_key": "Kopieren Sie diesen API-Schlüssel und speichern Sie ihn an einem sicheren Ort. Wenn Sie diesen Schlüssel verlieren, müssen Sie einen neuen generieren.",
-  "zapier_setup_instructions": "<0>Loggen Sie sich in Ihr Zapier-Konto ein und erstelle einen neuen Zap.</0><1>Wählen Sie Cal.com als ihre Trigger-App aus. Wählen Sie auch ein Auslöserereignis.</0><1>Wählen Sie ihr Konto aus und geben dann ihren eindeutigen API-Key ein.</2><3>Testen Sie den Trigger.</3><4>Alles fertig!</4>",
+  "zapier_setup_instructions": "<0>Loggen Sie sich in Ihr Zapier-Konto ein und erstelle einen neuen Zap.</0><1>Wählen Sie Cal.com als ihre Trigger-App aus. Wählen Sie auch ein Auslöserereignis.</0></1>Wählen Sie ihr Konto aus und geben dann ihren eindeutigen API-Key ein.</2><3>Testen Sie den Trigger.</3><4>Alles fertig!</4>",
```

Comment:
i used this regex to find these instances btw:

`(?:\\u003C|<)(\d+)(?:\\u003E|>)(.*?)(?:\\u003C|<)(\1)(?:\\u003E|>)`